platform-plugins.md

```diff
@@ -210,6 +210,59 @@ The iOS platform code of your plugin is located in `Pods/Development Pods/hello/
 
 You can run the example app by pressing the &#9654; button.
 
+#### iOS platform code (.swift)
+
+Flutter tooling currently expects iOS plugins to expose an Objective-C header. But you can
+implement your plugin in Swift instead and just have the Objective-C part forward plugin
+registration to that implementation. This section describes how to change the Objective-C
+plugin generated by `flutter create --plugin hello` to a Swift-based implementation.
+
+Before editing the iOS platform code in Xcode, first make sure that
+the code has been built at least once by Flutter by executing
+`cd hello/example; flutter build ios`.
+
+Next,
+
+1. Launch Xcode
+1. Select 'File > Open', and select the `hello/example/ios/Runner.xcworkspace` file.
+   The iOS platform code of your plugin is located in `Pods/Development Pods/hello/Classes/` in the Project Navigator.
+1. Select the `Classes` folder, then select `File > New > File...` and add a new Cocoa Touch class.
```

Comment:
Do we need the 'Cocoa Touch class' template? Looks like we are replacing all the contents anyway, so wouldn't the plain 'Swift class' work?